save-common/src/commonMain/kotlin/org/cqfn/save/entities/Annotations.kt

```diff
@@ -1,10 +1,13 @@
 package org.cqfn.save.entities
 
-@OptIn
+@OptionalExpectation
+@ExperimentalMultiplatform
```

Comment:
It's better to use `OptInt` with corresposing class here. This annotation isn't part of experimental API itself, but it requires it to operate.